pyrenew/latent/infectionswithfeedback.py

```diff
@@ -143,11 +143,18 @@ def sample(
         InfectionsWithFeedback
             Named tuple with "infections".
         """
-        if I0.size < gen_int.size:
+        if I0.shape[0] < gen_int.size:
```

Comment:
Multi-dim generation intervals now or out of scope?

Reply:
I would say out of scope for now as it's not required for site level wastewater model.